ui-components/src/forms/DOBField.stories.tsx

```diff
@@ -1,30 +1,25 @@
 import React from "react"
-
 import { DOBField } from "./DOBField"
-import { HouseholdMember } from "@bloom-housing/backend-core/types"
+import { useForm } from "react-hook-form"
```

Comment:
Since DOB Field requires a register and a watch, I used this here just to have it nicely and more realistically compile for the stories. I also don't think we should tie any component to react-hook-form.

Reply:
Ideally we wouldn't have react-hook-form in ui-component's package.json (I don't think it currently is), so will probably have to do without. If that makes implementing stories challenging let me know.

Reply:
I looked into this again and I couldn't get any of the form components to compile in the unit tests due to the way we're using the watch/register/error props without using react-hook-form, and we do import react-hook-form in PhoneField and TimeField in the component library already? Might be worth pairing on this?